devops/templates/requirements-installation-steps-template.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  pyVer:
+  requirementsFile:
+  pinRequirements:
+
+steps:
+- script: cp requirements.txt ${{requirementsFile}}
+    displayName: "Copy requirements file"
+    condition: ne('${{pyVer}}', '3.5')
```

Comment:
I hope we don't have to do too many of these. Have you also run the deployment pipeline (as far as Test-PyPI) to check this works?

Reply:
It seems inevitable to have many of these to be honest. Unless we're happy to drop py35

Reply:
The 'interesting' nature of conditional syntax in the templates certainly makes the concern greater. I agree that it's pretty much inevitable for now :-(

Did the release pipeline run OK with this in place?

Reply:
Notice that I've moved the logic into a python script instead because the conditions didn't evaluate the pyVer parameter. Apparently they only evaluate it during the execution but not the generation or at least that's what I saw a few people complain about. I don't like the condition syntax a lot anyway, so let me know how you feel about the script. I'll run the release pipeline as well, without releasing, though